### DISCUSSION TITLE
Using autoComplete.js in my WordPress Plugin

### DISCUSSION THREAD

**Discussion by LebCit** [TarekRaafat/autoComplete.js#166] [Show and tell]
Hello @TarekRaafat 

Just to say thank you very much for this beautiful piece of art.
I'm using it inside my WordPress plugin [FSMS](https://wordpress.org/plugins/full-screen-morphing-search/).
autoComplete.js is mentioned in the resources section.

Thanks again.
May THE CODE be with you.

SAY,
LebCit

**Comment by TarekRaafat** [comment]
Hello @LebCit,

Thank you so much glad you like it.
Great work! And thanks for keeping the credits; much appreciated. 

Let me know if you need any support.

Have a nice day. Cheers! :)
